### DISCUSSION TITLE
Changing write to update for `<C-s>` behavior?

### DISCUSSION THREAD

**Discussion by surmish** [LazyVim/LazyVim#9] [Ideas]
It avoids unneeded writes.

**Comment by folke** [comment]
Fixed!

**Comment by folke** [comment]
Changed it back to `write`, since it broke auto-formatting

**Comment by surmish** [comment]
Hey folke, what do you think about this? while trying to jump lines using count, it uses regular `j,k` behavior, `gj,gk` otherwise.

```
-- Remap for dealing with word wrap
vim.keymap.set("n", "j", "v:count == 0 ? 'gj' : 'j'", { expr = true, silent = true })
vim.keymap.set("n", "k", "v:count == 0 ? 'gk' : 'k'", { expr = true, silent = true })
```

**Reply by folke** [reply]
nice! Will add

**Reply by surmish** [reply]
Comment change
"Save in insert" Mode in key maps should change since it applies to more modes.

**Reply by folke** [reply]
@surmish what do you mean?

**Reply by surmish** [reply]
The comment here (config/keymaps.lua) should change since its not only insert mode any more  (i,v,n,s instead of just i)

```
-- save in insert mode
vim.keymap.set({ "i", "v", "n", "s" }, "<C-s>", "<cmd>w<cr><esc>", { desc = "Save file" })
```

**Reply by folke** [reply]
right, will update
